experiments/AtmosLES/stable_bl_model.jl

```diff
@@ -259,6 +260,9 @@ function stable_bl_model(
             (state, aux, t, normPu_int) -> C_drag,
             (state, aux, t) -> q_sfc,
         )
+        # elseif surface_flux == "Nishizawa2018"
+        #     energy_bc = PrescribedEnergyFlux((state, aux, t) -> LHF + SHF)
+        #     moisture_bc = PrescribedMoistureFlux((state, aux, t) -> moisture_flux)
```

Comment:
Any ideas on how to pass the boundary and first interior point states to the call that we would need here, @charleskawczynski ?

Reply:
I think we have two options:
 - add properties (`state_int`, `aux_int`) to `PrescribedEnergyFlux`, which we can then use in, for example, `src/Atmos/Model/bc_energy.jl`.
 - Update the anonymous function to also use `state_int`, `aux_int`. I.e., change `energy_bc = PrescribedEnergyFlux((state, aux, t) -> LHF + SHF)` to `energy_bc = PrescribedEnergyFlux((state, aux, t, state_int, aux_int) -> LHF + SHF)`
I think both will require changing `src/Atmos/Model/bc_energy.jl` and `src/Atmos/Model/bc_moisture.jl` etc.

Reply:
I think I'd probably lean towards the latter option